ciscoise.json

```diff
@@ -113,14 +119,13 @@
                         "d0337940-a86f-11e7-b6e9-000c29d5f0ea"
                     ],
                     "contains": [
-                        "ise endpoint id"
+                        "ise endpoint id",
+                        "ise resource id"
                     ]
                 },
                 {
                     "data_path": "action_result.data.*.SearchResult.resources.*.link.href",
                     "data_type": "string",
-                    "column_name": "Link to ISE Endpoint",
-                    "column_order": 2,
```

Comment:
Is there any specific reason for removing this from the output table?

Reply:
Removed this because the link in the column was broken. Was leading to nowhere